frontend/src/components/common/Platform/Card.vue

```diff
@@ -17,10 +17,10 @@ defineProps<{ platform: Platform }>();
       <v-card-text>
         <v-row class="pa-1 justify-center bg-primary">
           <div
-            :title="platform.name?.toString()"
+            :title="platform.name.toString()"
             class="px-2 text-truncate text-caption"
           >
-            <span>{{ platform.name }}</span>
+            <span>{{ platform.custom_name || platform.name }}</span>
```

Comment:
this can be done better with the pydantic resolver, i'll give it a crack tomorrow

Reply:
What do you mean? we still need to send both properties to show them in the platform info panel anyway